config/initializers/stripe.rb

```diff
@@ -1 +1,16 @@
-Stripe.api_key = ENV["STRIPE_SECRET_KEY"]
\ No newline at end of file
+Stripe.api_key = ENV["STRIPE_SECRET_KEY"]
+
+StripeEvent.configure do |events|
+  events.subscribe 'charge.succeeded' do |event|
+    Payment.create(
+      user: User.find_by(stripe_customer_id: event.data.object.customer),
+      amount: event.data.object.amount,
+      reference_id: event.data.object.id
+    )
+  end
+
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.